upgrading/upgrading_roxctl/upgrade-from-44.adoc

```diff
@@ -0,0 +1,74 @@
+[id="upgrade-from-44"]
+= Upgrading from version 3.0.44 or higher
+include::modules/common-attributes.adoc[]
+:context: upgrade-from-44
+
+toc::[]
+
+[role="_abstract"]
+Upgrade to the latest version of {product-title} from version 3.0.44 or higher.
+
+To upgrade {product-title} to the latest version, you must perform the following:
+
+* Backup the Central database
+* Upgrade the Central cluster
+* Upgrade all secured clusters
+
+include::modules/back-up-central-database.adoc[leveloffset=+1]
+
+[id="upgrade-central-cluster"]
+== Upgrading the Central cluster
+
+After you have backed up Central database, the next step is to upgrade Central and Scanner.
+
+include::modules/upgrade-central.adoc[leveloffset=+2]
+
+[id="upgrade-roxctl-cli"]
+=== Upgrading the `roxctl` CLI
+
+To upgrade the `roxctl` CLI to the latest version, download and install the latest version.
+
+include::modules/install-roxctl-cli-linux.adoc[leveloffset=+3]
+include::modules/install-roxctl-cli-macos.adoc[leveloffset=+3]
+include::modules/install-roxctl-cli-windows.adoc[leveloffset=+3]
+
+include::modules/upgrade-scanner.adoc[leveloffset=+2]
+
+include::modules/verify-central-cluster-upgrade.adoc[leveloffset=+2]
+
+[id="upgrade-secured-clusters"]
+== Upgrading all secured clusters
+
+After upgrading Central services, you must upgrade all secured clusters.
+
+[IMPORTANT]
+====
+* If you are using automatic upgrades:
+** Update all your secured clusters by using automatic upgrades.
+** Skip the instructions in this section and follow the instructions in the xref:#verify-last-check-in-time_{context}[Verifying last check-in time for secured clusters] and xref:revoke-the-api-token_{context}[Revoking the API token] sections.
```

Comment:
For these xrefs, you'll need to do the full path (xref:../../\<path>/<file_name.adoc>#\<anchor>).